programs/programs/policyengine/tests/test_policy_engine.py

```
@@ -0,0 +1,426 @@
+        self.assertEqual(len(marital_unit["members"]), 2)
+
+        # Should contain head and spouse IDs (order may vary)
+        member_ids = set(marital_unit["members"])
+        expected_ids = {str(self.head.id), str(self.spouse.id)}
+        self.assertEqual(member_ids, expected_ids)
+
+    def test_pe_input_populates_dependency_fields_for_txsnap(self):
+        """Test that TxSnap-specific dependencies populate correct fields."""
+        result = pe_input(self.screen, [TxSnap])
+        people = result["household"]["people"]
+        spm_unit = result["household"]["spm_units"]["spm_unit"]
```

Ruff (0.14.2)

Local variable `spm_unit` is assigned to but never used

Remove assignment to unused variable `spm_unit`

(F841)